apps/dashboard/src/locales/en/common/common.json

```diff
@@ -106,8 +106,10 @@
             },
             "error": {
                 "error": "Error",
-                "scanNotStarted": "NFC scan not started, please check permission regarding NFC.",
-                "unsupported": "Your browser is not mobile and does not have Web NFC."
+                "scanNotStarted": "NFC scan not started, please check permissions regarding NFC.",
+                "desktopUnsupported": "Browser is not mobile.",
```

Comment:
I would make this: NFC scan can only be done using a mobile phone.

That would make it more clear to the user what exactly the problem is